paasta_tools/cli/cmds/local_run.py

```diff
@@ -617,32 +624,33 @@ def configure_and_run_docker_container(
     Run Docker container by image hash with args set in command line.
     Function prints the output of run command in stdout.
     """
-    try:
-        system_paasta_config = load_system_paasta_config()
-    except PaastaNotConfiguredError:
-        sys.stdout.write(PaastaColors.yellow(
-            "Warning: Couldn't load config files from '/etc/paasta'. This indicates\n"
-            "PaaSTA is not configured locally on this host, and local-run may not behave\n"
-            "the same way it would behave on a server configured for PaaSTA.\n"
-        ))
-        system_paasta_config = SystemPaastaConfig({"volumes": []}, '/etc/paasta')
 
     soa_dir = args.yelpsoa_config_root
 
     volumes = list()
 
-    instance_type = validate_service_instance(service, instance, cluster, soa_dir)
-
     load_deployments = docker_hash is None or pull_image
 
+    interactive = args.interactive
+
     try:
-        instance_config = get_instance_config(
-            service=service,
-            instance=instance,
-            cluster=cluster,
-            load_deployments=load_deployments,
-            soa_dir=soa_dir,
-        )
+        if instance is None:
+            instance_type = 'adhoc'
+            instance = 'interactive'
+            instance_config = get_default_interactive_config(service=service, cluster=cluster, soa_dir=soa_dir)
+            interactive = True
+        else:
+            instance_type = validate_service_instance(service, instance, cluster, soa_dir)
+            instance_config = get_instance_config(
+                service=service,
+                instance=instance,
+                cluster=cluster,
+                load_deployments=load_deployments,
+                soa_dir=soa_dir,
+            )
+    except NoConfigurationForServiceError as e:
+        sys.stderr.write(str(e) + '\n')
+        return
```

Comment:
Nothing that calls this method is catching the return value and doing anything with it. I can change that?